forte/data/data_pack.py

```diff
@@ -245,18 +234,26 @@ def _validate(self, entry: EntryType) -> bool:
 
     @property
     def text(self) -> str:
-        r"""Return the text of the data pack"""
-        return self._text
+        """
+        Get text from a text payload at an index.
```

Comment:
This docstring exposes the implementation detail (payload), and what is `index` here?